app/controllers/idv/phone_controller.rb

```diff
@@ -8,6 +8,10 @@ class PhoneController < ApplicationController
     before_action :set_idv_form
 
     def new
+      if params[:location]
+        analytics.track_event(Analytics::IDV_PHONE_USE_DIFFERENT, location: params[:location])
```

Comment:
Elsewhere, we use "location" as a way to disambiguate different links within an individual step, whereas here it's a path. We also log "step" in a lot of cases too.

What do you think about logging `step` instead? Or something like `from_step`, if that's clearer?

There's a value in the view we use for cancel links:

https://github.com/18F/identity-idp/blob/7bc86dff4f4bd6b19f95dc6aabf19f2c163227d2/app/views/idv/otp_delivery_method/new.html.erb#L77

Reply:
Yeah, I was going to ask about that ;-)  updated in 76faec6